docs/translations/pt/scripting/callbacks/OnPlayerLeaveRaceCheckpoint.md

```diff
@@ -28,7 +28,7 @@ public OnPlayerLeaveRaceCheckpoint(playerid)
 
 ## Notes
 
-<TipNPCCallbacks />

```

Comment:
We can add a PT template for this.

Reply:
`id` (Indonesian) already has this, simply copy+paste the `id` directory here: https://github.com/openmultiplayer/web/tree/master/frontend/src/components/templates name it `pt` then change the content to:

```ts
export default function TipNpcCallback() {
  return (
    <Admonition type="tip">
      <p>Esta callback também pode ser chamada por um NPC.</p>
    </Admonition>
  );
}
```

And add to `index.ts`:

```ts
import VersionWarnPT from "./pt/version-warning";
import LowercaseNotePT from "./pt/lowercase-note";
import TipNPCCallbacksPT from "./pt/npc-callbacks-tip";
```